sdks/python/apache_beam/typehints/schemas.py

```diff
@@ -17,36 +17,36 @@
 
 """ Support for mapping python types to proto Schemas and back again.
 
-Python              Schema
-np.int8     <-----> BYTE
-np.int16    <-----> INT16
-np.int32    <-----> INT32
-np.int64    <-----> INT64
-int         ---/
-np.float32  <-----> FLOAT
-np.float64  <-----> DOUBLE
-float       ---/
-bool        <-----> BOOLEAN
-Timestamp   <-----> LogicalType(urn="beam:logical_type:micros_instant:v1")
-Mapping     <-----> MapType
-Sequence    <-----> ArrayType
-NamedTuple  <-----> RowType
-beam.Row    ---/
-
-nullable=True on a Beam FieldType is represented in Python by wrapping the
-typing in Optional.
-
-The mappings for STRING and BYTES are different between python 2 and python 3,
-because of the changes to str:
-py3:
-str/unicode <-----> STRING
-bytes       <-----> BYTES
-ByteString  ---/
-
-py2:
-str will be rejected since it is ambiguous.
-unicode     <-----> STRING
-ByteString  <-----> BYTES
+Imposes a mapping between common Python types and Beam portable schemas
+(https://s.apache.org/beam-schemas)::
+
+  Python              Schema
+  np.int8     <-----> BYTE
+  np.int16    <-----> INT16
+  np.int32    <-----> INT32
+  np.int64    <-----> INT64
+  int         ------> INT64
+  np.float32  <-----> FLOAT
+  np.float64  <-----> DOUBLE
+  float       ------> DOUBLE
+  bool        <-----> BOOLEAN
+  str/unicode <-----> STRING
+  bytes       <-----> BYTES
+  ByteString  ------> BYTES
+  Timestamp   <-----> LogicalType(urn="beam:logical_type:micros_instant:v1")
+  Mapping     <-----> MapType
+  Sequence    <-----> ArrayType
+  NamedTuple  <-----> RowType
+  beam.Row    ------> RowType
+
+Note that some of these mappings are provided as conveniences,
```

Comment:
Thanks for the note, this definitely helps.

Reply:
Thanks for helping me clarify my documentation :)